test/factories/api_docs/service.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+Factory.define(:api_docs_service, class: ApiDocs::Service) do |factory|
+  factory.sequence(:name) { |n| "service#{n}" }
```

Comment:
I pass the conversation from @hallelujah's comment from https://github.com/3scale/porta/pull/132#discussion_r225065675 to here
![image](https://user-images.githubusercontent.com/11318903/46942266-3c828700-d06d-11e8-9da5-38eb8aaa732a.png)

Reply:
No worries about this, if after this PR it still has that name, i will change it in the next PR about this mentioned in https://github.com/3scale/porta/pull/132#discussion_r225111927
But not urgent so don't worry about it 😉